src/core/execute.h

```diff
@@ -170,6 +170,7 @@ typedef struct ExecDirectory {
         mode_t mode;
         size_t n_items;
         ExecDirectoryItem *items;
+        uint32_t percent_quota;
```

Comment:
please make this:

```c
uint32_t quota_scale;    /* relative quota to backend size, scaled to 0…UINT32_MAX */
uint64_t quota_absolute; /* absolute quota in bytes */
```

and then fill in either depending on what the user actually specifies in the unit file.